ros2multicast/ros2multicast/api/__init__.py

```diff
@@ -19,8 +19,11 @@
 DEFAULT_PORT = 49150
 
 
-def send(data, *, group=DEFAULT_GROUP, port=DEFAULT_PORT):
+def send(data, *, group=DEFAULT_GROUP, port=DEFAULT_PORT, ttl=None):
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP)
+    if ttl is not None:
+        packed_ttl = struct.pack('b', ttl)
```

Comment:
Is this required for a particular platform?  I found some examples where the packing was not used.

Reply:
It is the recommended approach: https://docs.python.org/3/library/socket.html#socket.socket.setsockopt

An integer might work too. I just used the same approach as in the original PR.